### DISCUSSION TITLE
Allow Stonework Generator to ouput more than a single item

### DISCUSSION THREAD

**Discussion by MuteTiefling** [InnovativeOnlineIndustries/Industrial-Foregoing#1343] [Feature Requests]
For example, I'd like to make a recipe that crushes obsidian to 4x obsidian dust from Create. Or one that crushes netherrack to 2x cinder flour. Etc. 

Currently, it doesn't appear to be possible, as specifying a count in the JSON isn't respected. 

```js
        {
            output: { item: 'create:cinder_flour', count: 2 },
            input: { item: 'minecraft:netherrack' },
        }
```

**Comment by AiDSl** [comment]
only thing block this change is that it require work to write code instead just use vanilla Ingredient.
(Internally Stonework Generator is handled with Itemstack,which allow mutiple output)
